portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -52,7 +64,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the input from the comments form
     String authorText = getParameter(request, "author");
     String valueText = getParameter(request, "value");
-    comments.add(new Comment(authorText, valueText));
+    Comment comment = new Comment(authorText, valueText);
```

Comment:
If this value is special characters like ˙©ƒå∆ß˙©ƒ your site returns a 400. You should probably wrap all your database stuff in try/catch methods so that if the database fails for whatever reason, you can still display a usable webpage.